vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/accordion/AccordionWrapTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2022 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+package com.vaadin.flow.component.accordion;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.vaadin.flow.router.RouteConfiguration;
+import com.vaadin.testbench.unit.UIUnitTest;
+
+class AccordionWrapTest extends UIUnitTest {
+
+    @Override
+    protected String scanPackage() {
+        return "com.vaadin.flow.component.accordion";
+    }
+
+    AccordionView view;
+
+    @BeforeEach
+    public void registerView() {
+        RouteConfiguration.forApplicationScope()
+                .setAnnotatedRoute(AccordionView.class);
+        view = navigate(AccordionView.class);
+    }
+
+    @Test
+    void bySummary_returnsCorrectPanel() {
```

Comment:
Should test name be `getPanel_returnsCorrectPanel`?

Reply:
changed to getPanelBySummary_returnsCorrectPanel